spec/services/ccms/applicant_add_requestor_spec.rb

```diff
@@ -21,7 +21,9 @@ module CCMS
              date_of_birth: Date.new(1969, 1, 1))
     end
 
-    let(:requestor) { described_class.new(applicant) }
+    let(:provider_username) { 'my_login' }
+
+    let(:requestor) { described_class.new(applicant, provider_username) }
 
```

Comment:
As the `provider_username` is only used in the requestor, couldn't we just hardcode it in the argument to avoid the indirection of the `let` declaration?
If we are worried on the argument meaning we could even use `"provider username"` as the  value itself.

Reply:
True. I'll change it in all the specs 👍